packages/tools/Makefile

```
@@ -701,6 +704,36 @@ SCW-instance-delete:
+			--arg prefix "${SCW_STALE_NAME_PREFIX}" \
+			--argjson now "$${NOW_EPOCH}" \
+			--argjson max_age "${SCW_STALE_MAX_AGE_SECONDS}" \
+			'.servers[] \
+			| select(.project == $$project) \
+			| select(.state == "running") \
+			| select((.name | ascii_downcase) | startswith($$prefix | ascii_downcase)) \
+			| select(($$now - ((.creation_date // .created_at) | sub("\\.[0-9]+Z$$"; "Z") | fromdateiso8601)) > $$max_age) \
+			| .id' \
+		> ${CLOUD_STALE_RUNNING_IDS_FILE}
+
+SCW-instance-delete-stale-running: SCW-instance-get-stale-running-ids
```

checkmake (0.3.2)

[warning] Target body for "SCW-instance-delete-stale-running" exceeds allowed length of 5 lines (11).

(maxbodylength)